elasticdl/python/common/k8s_client.py

```diff
@@ -183,20 +183,33 @@ def _create_pod(self, **kargs):
 
         # Mount data path
         if kargs["volume"]:
-            volume_dict = parse_volume(kargs["volume"])
-            volume_name = kargs["pod_name"] + "-volume"
-            volume = client.V1Volume(
-                name=volume_name,
-                persistent_volume_claim=V1PersistentVolumeClaimVolumeSource(
-                    claim_name=volume_dict["claim_name"], read_only=False
-                ),
-            )
-            spec.volumes = [volume]
-            container.volume_mounts = [
-                client.V1VolumeMount(
-                    name=volume_name, mount_path=volume_dict["mount_path"]
+            spec.volumes = []
+            container.volume_mounts = []
+            volume_dicts = parse_volume(kargs["volume"])
+            for i, volume_dict in enumerate(volume_dicts):
```

Comment:
what does i mean? volume index?

Reply:
Yeah, we need to name the volume with the index.